### PR TITLE
JDK-8312049: runtime/logging/ClassLoadUnloadTest can be improved

### PR DESCRIPTION
**Pull request by tstuefe**
Trivial improvements, mainly not spawning individual child processes for every call to `checkFor` and `checkAbsent`. Runtime goes down from 14s -> 10s. We could reduce the runtime a lot more by splitting up the test but I avoided that.

Checked manually on x64 Linux.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8312049](https://bugs.openjdk.org/browse/JDK-8312049): runtime/logging/ClassLoadUnloadTest can be improved (**Enhancement** - P4)

### Reviewers
 * [Doug Simon](https://openjdk.org/census#dnsimon) (@dougxc - Committer)
 * [David Holmes](https://openjdk.org/census#dholmes) (@dholmes-ora - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/14876/head:pull/14876` \
`$ git checkout pull/14876`

Update a local copy of the PR: \
`$ git checkout pull/14876` \
`$ git pull https://git.openjdk.org/jdk.git pull/14876/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 14876`

View PR using the GUI difftool: \
`$ git pr show -t 14876`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/14876.diff">https://git.openjdk.org/jdk/pull/14876.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/14876#issuecomment-1634735406)

### CODE REVIEW

**Comment by tstuefe** [comment]
Pinging @dougxc

**Review by dougxc — APPROVED** [review]

**Comment by tstuefe** [comment]
Thanks @dougxc !

Not sure if this counts as trivial enough, better get a second review. @MBaesken maybe?

**Review by dholmes-ora — APPROVED** [review]
Looks good - thanks.

P.S. No not "trivial" :)

**Comment by tstuefe** [comment]
Thank @dholmes-ora !

/integrate

**Pull request closed without merging**
